### PR TITLE
doc update front page and side bar

### PR DESCRIPTION
**Pull request by SUGURUSEO**
1. Added "About Astar" and "Ecosystem"
2. Updated titles and order to be the same in the side manu and the front page
3. Added youtube link on the top bar

Detail is explained in this [notion](https://www.notion.so/astarnetwork/Adding-missing-titles-in-the-Docs-43f0fd2f6e134fa4ac9b07143fb865f5)

### CODE REVIEW

**Comment by Kahonnohak** [comment]
@SUGURUSEO Thank you so much! 
I added @fiexer and @impelcrypto as reviewers.

**Comment by SUGURUSEO** [comment]
Thanks @Kahonnohak !
Now icons are updated as well for "About Astar" and "Ecosystem". 
So everything is ready.

**Review by impelcrypto — APPROVED** [review]

File: src/components/HomepageFeatures/index.tsx [hunk 1]
```diff
@@ -10,6 +10,16 @@ type FeatureItem = {
 };
 
 const FeatureList: FeatureItem[] = [
+  {
+    title: 'About Astar',
+    link: '/docs/about/astar',
+    iconClass: 'aboutastar',
+    description: (
+      <>
+        This section gives you a brief idea of the problems Astar solves and our token economics including token allocation.
```
Comment (fiexer) [inline src/components/HomepageFeatures/index.tsx:19]: Please rewrite.

File: src/components/HomepageFeatures/index.tsx [hunk 2]
```diff
@@ -21,6 +31,27 @@ const FeatureList: FeatureItem[] = [
       </>
     ),
   },
+  {
+    title: 'Ecosystem',
+    link: '/docs/ecosystem/',
+    iconClass: 'ecosystem',
+    description: (
+      <>
+        This section gives you an overview of our ecosystem, such as Polkadot, Substrate, and Initiatives.
```
Comment (fiexer) [inline src/components/HomepageFeatures/index.tsx:40]: Please rewrite.

File: src/components/HomepageFeatures/index.tsx (hunk 1 above)
Comment (fiexer) [inline src/components/HomepageFeatures/index.tsx:19]: Rewrite

File: src/components/HomepageFeatures/index.tsx (hunk 2 above)
Comment (fiexer) [inline src/components/HomepageFeatures/index.tsx:40]: Rewrite

**Review by fiexer — CHANGES_REQUESTED** [review]

File: src/components/HomepageFeatures/index.tsx (hunk 1 above)
Comment (SUGURUSEO) [reply src/components/HomepageFeatures/index.tsx]: Updated

File: src/components/HomepageFeatures/index.tsx (hunk 2 above)
Comment (SUGURUSEO) [reply src/components/HomepageFeatures/index.tsx]: updated

**Comment by SUGURUSEO** [comment]
@fiexer Please review this PR when you get time. The order is updated as well as description and reviewed by the marketing team. Thanks!

**Review by fiexer — APPROVED** [review]

**Review by Kahonnohak — APPROVED** [review]
LGTM!

**Pull request merged**